src/test/java/com/hubspot/jinjava/tree/parse/CustomTokenScannerSymbolsTest.java

```diff
@@ -0,0 +1,105 @@
+package com.hubspot.jinjava.tree.parse;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.Lists;
+import com.hubspot.jinjava.Jinjava;
+import com.hubspot.jinjava.JinjavaConfig;
+import com.hubspot.jinjava.lib.filter.JoinFilterTest.User;
+import java.util.HashMap;
+import org.junit.Before;
+import org.junit.Test;
+
+public class CustomTokenScannerSymbolsTest {
+  private Jinjava jinjava;
+  private JinjavaConfig config;
+
+  @Before
+  public void setup() {
+    config =
+      JinjavaConfig.newBuilder().withTokenScannerSymbols(new CustomTokens()).build();
+    jinjava = new Jinjava(config);
+    jinjava.getGlobalContext().put("numbers", Lists.newArrayList(1L, 2L, 3L, 4L, 5L));
+  }
+
+  @Test
+  public void testsThatCustomTokensDoesNotFail() {
```

Comment:
can you make these rspec-style? `itRendersWithCustomTokens`, etc.

Reply:
sure.